torch/CMakeLists.txt

```diff
@@ -412,6 +420,8 @@ endif()
 target_compile_definitions(torch_python PRIVATE "-DTHP_BUILD_MAIN_LIB")
 
 target_link_libraries(torch_python PRIVATE ${TORCH_LIB} ${TORCH_PYTHON_LINK_LIBRARIES})
+target_link_libraries(torch_python PRIVATE torch::xpurt)
+target_link_libraries(torch_python PRIVATE c10_xpu)
```

Comment:
should we append xpurt and c10_xpu into TORCH_PYTHON_LINK_LIBRARIES in somewhere, instead of specifying them here? How does CUDA do?